### PR TITLE
 dev-python/line_profiler: Version bump

### PR DESCRIPTION
**Pull request by marbre**

### CODE REVIEW

File: dev-python/line_profiler/metadata.xml [hunk 1]
```diff
@@ -9,6 +9,10 @@
     <email>sci@gentoo.org</email>
     <name>Gentoo Science Project</name>
   </maintainer>
+  <maintainer type="project">
+    <email>proxy-maint@gentoo.org</email>
+    <name>Gentoo Proxy Maintainers Project</name>
```
Comment (mgorny) [inline dev-python/line_profiler/metadata.xml]: A project indeed, so why do you call us 'herd' in the commit message? Do we look like sheep to you? ;-P

File: dev-python/line_profiler/line_profiler-2.1.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+KEYWORDS="~amd64"
+
+SLOT="0"
+LICENSE="BSD"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="dev-python/ipython[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}
+	test? ( dev-python/pytest[${PYTHON_USEDEP}] )
```
Comment (mgorny) [inline dev-python/line_profiler/line_profiler-2.1.2.ebuild]: Why? I don't see it being used anywhere.

File: dev-python/line_profiler/line_profiler-2.1.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	https://pypi.python.org/pypi/line_profiler"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+KEYWORDS="~amd64"
+
+SLOT="0"
+LICENSE="BSD"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="dev-python/ipython[${PYTHON_USEDEP}]"
```
Comment (mgorny) [inline dev-python/line_profiler/line_profiler-2.1.2.ebuild]: Also you're missing RDEP+DEP on setuptools (it uses entry_points).

File: dev-python/line_profiler/metadata.xml (hunk 1 above)
Comment (marbre) [reply dev-python/line_profiler/metadata.xml]: Indeed. Fixed the commit message.

**Comment by marbre** [comment]
@mgorny Thanks for the review. Fixed the deps.

**Comment by mgorny** [comment]
Thanks. Looks good, passes tests, I'll merge it now.

**Pull request closed without merging**